components/database/sqlite/manifests/sample-manifest.p5m

```diff
@@ -30,6 +30,8 @@ link path=usr/lib/$(MACH64)/libsqlite3.so target=libsqlite3.so.0.8.6
 link path=usr/lib/$(MACH64)/libsqlite3.so.0 target=libsqlite3.so.0.8.6
 file path=usr/lib/$(MACH64)/libsqlite3.so.0.8.6
 file path=usr/lib/$(MACH64)/pkgconfig/sqlite3.pc
+file path=usr/lib/$(MACH64)/sqlite3/libtclsqlite3.so
```

Comment:
Should we ship it?

Reply:
We can ship it, but I never managed to install 32-bit `libtclsqlite3.so`, it gets overwritten by 64-bit one. I can include the 64-bit one and the associated `pkgIndex.tcl` file, but I would not bother fixing the 32-bit one. I don't know how it relates to `runtime/tcl-8/tcl-sqlite-3`.